guide/english/computer-science/dynamic-programming/index.md

```diff
@@ -15,34 +15,37 @@ This is the tree to find F(5):
 
 ![Fibonacci serie's tree](https://cdn-media-1.freecodecamp.org/imgr/59Rpw.png)
 
-To compute F(5) it will need to compute many times the same F(i). Using recursion:
+To compute F(5), the recursive method will compute the same value multiple times
 
+For example:
+
+F(0) is called 3 times </br>
```

Comment:
Please remove the line breaks. The way we display articles, they’ll get added automatically

Reply:
Done!